README.md

```diff
@@ -43,9 +43,9 @@ yarn install
 
 ### **Configuring a Mac Dev Environment**
 
-**Required Tools:** [Android Studio](https://developer.android.com/studio/) (including NDK 21.0.6113669), [CMake](https://cmake.org/)
+**Required Tools:** [Android Studio](https://developer.android.com/studio/) (including NDK 21.0.6113669), [CMake](https://cmake.org/), [Ninja](https://ninja-build.org/)
 
-- The `PATH` environment variable must include the path to adb (typically ~/Library/Android/sdk/platform-tools/).
+- The `PATH` environment variable must include the path to adb (typically ~/Library/Android/sdk/platform-tools/) and the path to the extracted Ninja binary (extracted from the downloaded zip file).
```

Comment:
I would put this on another line, and say something like:
- The `PATH` environment variable must include the path to Ninja, or Ninja must be [installed via a package manager](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages).

Reply:
Will do. Thanks for the comment.